### ISSUE TITLE
[bug] The tokenizer in the tutorial doesn't work

### ISSUE THREAD

**Issue by Lokiare** [huggingface/sharp-transformers#4]
The tokenizer in the tutorial:
SentenceSimiliarityUtils_.TokenizeInput(input);

Visual Studio gives a not-defined error for SentenceSimiliarityUtils_.

**Comment by li-sicong** [comment]
Those utils are in the demo itself, not part of this package: https://drive.google.com/drive/folders/1LP109dhRjniXdLrekzgr6ld5HEXczoJU

**Comment by IsaacGreenMachine** [comment]
yup same issue here. I'd love to build the project from scratch - where is the `SentenceSimilarityUtils` namespace used in the final project's `SentenceSimilarity.cs` coming from?

**Comment by simoninithomas** [comment]
Hey there 👋 , which version of the tutorial are you using? This version? https://huggingface.co/learn/ml-games-course/unit1/introduction
